tests/test_info_server.py

```diff
@@ -43,6 +45,7 @@ def test_success_write_info(self):
         self.assertEqual(file_data["metadata"]["CPU_LIMIT"], "3")
         self.assertEqual(file_data["protocol"], "uds")
         self.assertEqual(file_data["language"], "python")
+        self.assertEqual(file_data["minimum_client_version"], ">= 1.2.0-0")
```

Comment:
nit: use `MINIMUM_CLIENT_VERSION` otherwise we need to update the unit test every time we update `MINIMUM_CLIENT_VERSION`.